pyincore/client.py

```diff
@@ -16,6 +18,66 @@
 logger = pyglobals.LOGGER
 
 
+def edit_hash_entry(mode, hashed_url=None, service_url=None):
```

Comment:
minor nitpick, but should we call this method update_hash_entry?

Reply:
Thoughts @nunezco2 @longshuicy?

Reply:
`upload` sounds good to me.

Reply:
What should I change the name to then?

Reply:
`update_hash_entry`